### PR TITLE
use variable instead of literal in server.xml

### PR DESCRIPTION
**Pull request by dacron**
#### Pull Request (PR) description
Using the `proxy` parameter when `tomcat_native_ssl` is true results in a string literal of "key" in server.xml in the SSL connector.

Pull request address that by using the previously declared `$key` variable in the EPP template.

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Comment by dacron** [comment]
@bastelfreak this test failure doesn't look related to the change?

**Comment by bastelfreak** [comment]
that's the kind of error you get if docker failed. I restarted the jobs.

File: templates/server.xml.epp [hunk 1]
```diff
@@ -87,7 +87,7 @@
                     keystoreType="<%= $jira::tomcat_keystore_type %>"
 <%   if $jira::proxy { -%>
 <%     jira::sort_hash($jira::proxy).each |$key, $value| { -%>
-                    <%= key %> = '<%= $value %>'
+                    <%= $key %> = '<%= $value %>'
```
Comment (bastelfreak) [inline templates/server.xml.epp:90]: that really looks like a bug that should have been catched by the unit tests. @dacron could you add a test for this that verifies the xml content?

Comment (dacron) [reply templates/server.xml.epp:90]: I've added a test.

**Pull request merged**